src/lib/chips/chip-list.ts

```diff
@@ -49,69 +49,125 @@ import {SPACE, LEFT_ARROW, RIGHT_ARROW} from '../core/keyboard/keycodes';
 })
 export class MdChipList implements AfterContentInit {
 
-  /** Track which chips we're listening to for focus/destruction. */
-  private _subscribed: WeakMap<MdChip, boolean> = new WeakMap();
+  /** Whether or not the chip list is currently focusable via keyboard interaction. */
+  protected _tabIndex = -1;
+
+  /** When a chip is destroyed, we track the index so we can focus the appropriate next chip. */
+  protected _destroyedIndex: number = null;
 
-  /** Whether or not the chip is selectable. */
-  protected _selectable: boolean = true;
+  /** Track which chips we're listening to for focus/destruction. */
+  protected _subscribed: WeakMap<MdChip, boolean> = new WeakMap();
 
   /** The ListKeyManager which handles focus. */
   _keyManager: ListKeyManager;
 
   /** The chip components contained within this chip list. */
   chips: QueryList<MdChip>;
 
-  constructor(private _elementRef: ElementRef) { }
+  constructor(protected _renderer: Renderer, protected _elementRef: ElementRef,
+              @Optional() protected _mdInputContainer: MdInputContainer) {
```

Comment:
So far we've been trying to avoid introducing any inter-component dependencies. In this case, I don't think we want to tightly couple the md-input with the chip-list, since people may use chips in combination with their own custom input (which is something Jen's team wants to do). 

A less-coupled approach would be to create an interface like `ChipInput`, and accept that as an `@Input` (and making sure that either `MdInput` or `MdInputContainer` conforms to that interface. Then we could connect the two without introducing a dependency.